test/autoscaling.test.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const assume = require('assume');
+const async = require('async');
+const nock = require('nock');
+
+const common = require('./common');
+const beforeTesting = common.beforeTesting;
+const resourceName = 'test-name';
+
+describe('lib.autoscaling', () => {
+  describe('.autoscaling', () => {
+    const path = `/apis/autoscaling/v1/namespaces/${ common.currentName }/horizontalpodautoscalers`;
+    const resourcePath = `${ path }/${ resourceName }`;
+    const horizontalAutoscaleObj = {
+      apiVersion: 'autoscaling/v1',
+      kind: 'HorizontalPodAutoscaler',
+      metadata: {
+        name: 'test-name',
+        namespace: 'fa'
+      },
+      spec: {
+        targetCPUUtilizationPercentage: 65,
+        maxReplicas: 10,
+        minReplicas: 3,
+        scaleTargetRef: {
+          apiVersion: 'autoscaling/v1beta1',
+          kind: 'Deployment',
+          name: 'example-name',
+          subresource: 'scale'
+        }
+      },
+      status: {
+        currentReplicas: 0,
+        desiredReplicas: 0
+      }
+    };
+
+    beforeTesting('int', common.changeName);
+    beforeTesting('unit', () => {
+      const mockDs = {
+        apiVersion: 'autoscaling/v1',
+        kind: 'HorizontalPodAutoscaler',
+        metadata: {
+          name: 'test-name',
+          namespace: 'example-namepace'
+        },
+        spec: {
+          targetCPUUtilizationPercentage: 65,
+          maxReplicas: 10,
+          minReplicas: 3,
+          scaleTargetRef: {
+            apiVersion: 'autoscaling/v1beta1',
+            kind: 'Deployment',
+            name: 'example-name',
+            subresource: 'scale'
+          }
+        },
+        status: {
+          currentReplicas: 0,
+          desiredReplicas: 0
+        }
+      };
+      nock(common.autoscaling.url)
+                .post(path)
```

Comment:
This indent looks off.  I would have expected two spaces from `nock` statement:

```js
nock(common.autoscaling.url)
  .post(path)
  .reply(201, mockDs)
  ...
```

Can you adjust it?

I'm also surprised `npm run lint` or `npm run test` didn't complain, but that's a different matter.

Reply:
Formatting updated, npm lint shows the following output...

```
/usr/local/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js run lint --scripts-prepend-node-path=auto

> kubernetes-client@3.16.0 lint /Users/darrenackers/Projects/ThirdParty/kubernetes-client
> eslint-godaddy lib/ test/


Process finished with exit code 0
```

Potentially an issue with my machine?